### ISSUE TITLE
KeyError: 'sentry.interfaces.Exception'

### ISSUE THREAD

**Issue by evz** [datamade/semabot#9]
Sometimes we don't log exceptions. Sometimes we log messages so the sentry interface is not the exception interface. Let's check for which interface is being used so that we can avoid these silly exceptions.

https://sentry.io/datamade/semabot/issues/344704948/

```
KeyError: 'sentry.interfaces.Exception'
(2 additional frame(s) were not displayed)
...
  File "semabot-d-E199FXLIO/webhooks.py", line 82, in sentry
    exception = data['event']['sentry.interfaces.Exception']

KeyError: 'sentry.interfaces.Exception'
```

**Comment by evz** [comment]
This is an example of a message getting logged: https://sentry.io/datamade/dedupe-api/issues/349196309/

**Comment by evz** [comment]
closed with 7bcf43becd89299704e9660b6374d46f5912afc4
